### DISCUSSION TITLE
Diabling Process/services after installation in distributed setup

### DISCUSSION THREAD

**Discussion by chandieka** [Security-Onion-Solutions/securityonion#1895] [Unsupported Versions] [answered]
Hi!, what is the recommended action? if let's say I want to disable (permanently) a certain service on the sensor such as the "full packet capture" after the installation of the environment so that whenever the services are restarted it won't turn itself on. 

it was fairly simple on version 16.04 and it was documented [here](https://docs.securityonion.net/en/16.04/disabling.html).

now that version 2.x is using Saltstack is there an equivalent action that can be done through the manager node?

Thanks in advance.

**Comment by shipler** [comment, marked as answer]
you can modify the global.sls (or minions sls) on the master, there's a sensor section and below there's an enabled option.

> /opt/so/saltsatck/local/pillar/global.sls
> /opt/so/saltsatck/local/pillar/minions/

Regarding the full packet capture it will be supported in next release:
https://github.com/Security-Onion-Solutions/securityonion/issues/1601
